packages/zowe-explorer/src/utils/ProfilesUtils.ts

```diff
@@ -142,7 +142,8 @@ export async function isUsingTokenAuth(profileName: string): Promise<boolean> {
     const secureProfileProps = await Profiles.getInstance().getSecurePropsForProfile(profileName);
     const secureBaseProfileProps = await Profiles.getInstance().getSecurePropsForProfile(baseProfile?.name);
     if (isUsingZosmf && baseProfile) {
```

Comment:
This `if(iszosmf && baseProfile)` will cause issues if another profile type is registered with API ML and using baseProfile to hold the "tokenType": "apimlAuthenticationToken" with tokenValue.

Reply:
could possibly simplify the check for all profile types, idk still testing it.
```
const baseProfile = Profiles.getInstance().getDefaultProfile("base");
const secureProfileProps = await Profiles.getInstance().getSecurePropsForProfile(profileName);
const secureBaseProfileProps = await Profiles.getInstance().getSecurePropsForProfile(baseProfile?.name);
const profileUsesBasicAuth = secureProfileProps.includes("user") && secureProfileProps.includes("password");
return (secureProfileProps.includes("tokenValue") || secureBaseProfileProps.includes("tokenValue")) && !profileUsesBasicAuth;
```